modules/UI/videolayout/SmallVideo.js

```diff
@@ -652,21 +655,42 @@ SmallVideo.prototype.updateView = function () {
                                 || displayMode === DISPLAY_VIDEO_WITH_NAME));
 };
 
+/**
+ * Updates the react component displaying the avatar with the passed in avatar
+ * url.
+ *
+ * @param {string} avatarUrl - The uri to the avatar image.
+ * @returns {void}
+ */
 SmallVideo.prototype.avatarChanged = function (avatarUrl) {
-    var thumbnail = $('#' + this.videoSpanId);
-    var avatarSel = this.$avatar();
+    const $thumbnail = this.$avatar();
+    const thumbnail = $thumbnail && $thumbnail.get(0);
```

Comment:
They're different but end in the same result. Assuming this.$avatar() is always a jquery lookup, $thumbnail will always be defined but may not have any elements in its internal array of found elements. If it has no elements, the get(0) call will return undefined anyway. This makes me think the truthy check for $thumbnail isn't even needed so I'll get rid of it.